modules/microshift-nw-config-route-admission.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-nw-router.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-route-admission_{context}"]
+= Configuring the route admission policy
+
+By default, {microshift-short} allows routes in multiple namespaces to use the same hostname. To prevent routes in different namespaces from claiming the same hostname, you can configure the value.
+
+.Prerequisites
+
+* You created a {microshift-short} `config.yaml` file.
+* The {oc-first} is installed.
+
+.Procedure
+
+. To prevent routes in different namespaces from claiming the same hostname, update the `namespaceOwnership` field value to `Strict` in the {microshift-short} `config.yaml` file as shown in the following example:
++
+.Example `config.yaml` route admission policy
+[source,yaml]
+----
+# ...
+ingress:
+  routeAdmissionPolicy:
+    namespaceOwnership: Strict <1>
+# ...
+----
+<1> Valid values are `Strict` and `InterNamespaceAllowed`. Specifying `Strict` prevents routes in different namespaces from claiming the same host. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
++
+[TIP]
+====
+If you complete all the configurations that you need to make in the {microshift-short} `config.yaml` file at the same time, you can minimize system restarts.
```

Comment:
```suggestion
If you complete the `config.yaml` file configuration at the same time, you can minimize system restarts.
```

Reply:
Yes, that is the meaning. But I am retaining this Tip as is to maintain consistency with this https://github.com/openshift/openshift-docs/pull/76100 PR.

Reply:
Would you consider rephrasing these `[TIP]` instances in a separate follow up PR (or in this PR)? Because, right now given the context of this section, the tip is not meaningful. 
User might miss this important detail if not rephrased.

Reply:
You can maintain the consistency by rephrasing all similar occurrences of TIP.

Reply:
As per our discussion retaining this TIP as is.

Reply:
We could do something like:
You can minimize system restarts by planning the configuration needs for your use case in advance and then by making all of the necessary changes to the {microshift-short} `config.yaml` file at the same time.

Reply:
making this change as per discussion with Shauna.